### ISSUE TITLE
Designing a new logo for Wordplay

### ISSUE THREAD

**Issue by YizhouDing** [wordplaydev/wordplay#414]
## What's the problem?

-   What is the task, goal, or need this request would help someone accomplish?
 A new logo would help to strengthen our brand image and improve recognition.

-   What is the context or scenario in which they would experience it?
Users encounter our logo across various platforms such as our website, social media profiles, marketing materials, and products. It's crucial that our logo communicates our brand message effectively in all these contexts.

-   Who is the group of people who would be in this situation?
Our target audience includes potential customers, current users, and anyone interacting with our brand across different touchpoints.

-   What is it about the current design that makes the need above hard to satisfy?
Our logo is a chat bubble, even though it is accessible for screenreaders, however, it lacks uniqueness, clarity, or relevance to our brand, making it difficult to leave a memorable impression on our audience.

## What's the design idea?

What do you propose to change or add to address the problem above? Describe some functionality, a design concept, a redesign, or alternative designs that might address the problem. Provide as much detail about the idea as you currently have, but no need to provide an exact design specification if you don't have one. (That goes below).

- Opt for a minimalist design that is clean, simple, and easily recognizable. This ensures versatility and scalability across different platforms and sizes.
- Ensure that the logo design is scalable without losing clarity or legibility. It should also be adaptable to different backgrounds and contexts

## Design specification

_Reporters can leave this blank. Designers should write a detailed, precise description of what is to be built. If images are necessary to convey this precisely, make sure to include image descriptions, so everyone can see what is in the image. Do not link to external documents; everything should be embedded here. Once the design is approved, we will remove the `needs design` tag and replace it with the `buildable` tag, signalling that it can be built._

**Comment by YizhouDing** [comment]
@amyjko Hi Amy, Based on our discussion last week, I created a new issue. Please let me know what else needs to be improved about this issue.

**Comment by amyjko** [comment]
Thanks for reporting @YizhouDing! I just made one tweak to description of the group (we don't have customers, but we do have a very specific target population).

**Comment by kellyjhorak** [comment]
Hi @amyjko , I would love to work on this. How can I get assigned?

**Comment by amyjko** [comment]
@kellyjhorak I assigned you.

**Comment by patriciacindy** [comment]
Hello, I am interested to work on this project. Would you guys mind if I join? @kellyjhorak @sanambk @tshres

**Comment by sanambk** [comment]
Hi, yes no problem at all! Did you want to work with us? If so, we are in
the front left corner - I’m wearing a grey coffee shop hoodie and my hair
up in a bun - in case you wanted to join us!

Sanam

**Comment by patriciacindy** [comment]
@sanambk  Yes, I would like to discuss the progress on Discord. My discord name is patriciacindyisabell. I have messaged you. We can create a group on Discord to communicate.

**Comment by sanambk** [comment]
[Logo Iteration Documentation.pdf](https://github.com/wordplaydev/wordplay/files/15488749/Logo.Iteration.Documentation.pdf)

Draft iterations with @tshres and @kellyjhorak

**Comment by zihanRenL** [comment]
Hi I want to be assigned to this issue! Is there any size requirement for the logo? (like 500 * 500px?

**Comment by zihanRenL** [comment]
Hi professor I had a few drafts (currently using the brush in Adobe Illustrator and will further revise and refine using pen based on your feedback). I looked up the previous document for logo iteration and came up with:
1. world-related
An icon of earth where the continents are replaced by interlocking shapes resembling two hands grasping each other, symbolizing unity, collaboration, and inclusivity. Surrounding the Earth are various programming-related icons, such as mathematical functions (f), curly braces ({}), and symbols for sound and playback (a speaker and a play button), representing the coding focus of Wordplay. Also including the icon of screen readers for accessibilities
![image](https://github.com/user-attachments/assets/08e55d07-b60c-4c92-a462-ae24dc797f7a)
2. puzzle-related
(1)Use puzzle to forms an image of the Earth or to form a speech bubble, can add other coding-related element insides the puzzle. The one with the speech bubble also includes a speaker, representing the accessibility features of screen readers.
![image](https://github.com/user-attachments/assets/5b52761d-1f90-47e6-8d2e-3cd424ce66ce)
(2)Use puzzle where each piece has rounded indentations and protrusions representing different people. The individuals depicted in the puzzle may have different physical disabilities, and they are all able to access programming through our accessible features, such as keyboards and screen readers.
![image](https://github.com/user-attachments/assets/c4725ec0-0d9e-41d9-914f-6208b4a9120e)
3. Light Bulb and education related
Use the light bulb from our website, but instead of the filament, include an image of a tree intertwined with children figures that also have branches. The base of the bulb is represented by three books, symbolizing education.
![image](https://github.com/user-attachments/assets/66d9dc1e-79c2-487d-9870-2b8afd31b181)

One of the worries I had might be, I noticed that the puzzle piece is frequently used in contexts related to autism, which makes me wonder if using a puzzle for our logo might lead to confusion and reduce its unique recognizability, but I also feel that the puzzle element is very effective at representing the diversity of the world and the idea of inclusion...

**Comment by amyjko** [comment]
I love the design exploration! Various reactions:

* The puzzle is an interesting metaphor. It has the issues you mentioned, but also other issues: it implies there's one correct solution and that everyone makes the same thing, but that's the opposite of what the platform is: everyone makes different things on this platform.

* Most of the concepts have a lot of detail, which doesn't translate well to small 64x64 icon formats for the logo (but would look fine larger). I think you'll have to explore logos that are legible at low levels of detail.

* I think your explorations do a good job engaging things like learning, inclusion, and ability diversity. I wonder what other visual metaphors might capture diversity in ability and culture, while also linking to programming? I wonder if you could explore things that are more typographic metaphors, since the platform is very much about language and text.

**Comment by zihanRenL** [comment]
![wordplayDec9_画板 1](https://github.com/user-attachments/assets/1b7ea04b-f2c8-424f-9e29-9dfac3cd60ce)
I tried to make them simpler and less detailed based on your feedback. I tried to make it linked to programming like the top-left one with <> and the bottom-left one with placeholder and carnet down in our UI. Also I explored some multilingual elements. The quotation marks in the top-right corner reflect the typographic styles used in different languages.
I want to attach the .ai file but Github doesn't allow me to do that, are there any other way to do that or just .png is fine?

**Comment by amyjko** [comment]
(You should be able to attach files with the attach button in the toolbar.)

It's an interesting design exploration! They all convey very different emotions. What are the concepts you were going for with each? One way to define these is to write the text description of them that you would use for alt text for screen reader users. What is each trying to communicate?

**Comment by amyjko** [comment]
Once a quarter, I check each assignment to check the plans of the people that are assigned an issue. Please reply and indicate whether you intend to continue work on this issue. If you do not, please assign assign yourself, so we have an accurate account of who is working on an issue.

**Comment by amyjko** [comment]
No reply from @zihanRenL, unassigning.

**Comment by nyechuri** [comment]
hi! I would love to work on this! I looked at the bar on the side and noticed that there were no assignees so could I get added to this!

**Comment by amyjko** [comment]
@nyechuri Assigned!

**Comment by amyjko** [comment]
@nyechuri Are you still working on this? I know it was a recent assignment, but we're doing quarterly cleanup.

**Comment by rameenniazii95-star** [comment]
Hi Amy, this looks fun can you assign me this and i asked you about how many points does this get me on discord?

**Comment by amyjko** [comment]
Yes, I've added you. As noted in the INFO 494 syllabus, design contributions are effort based.

Logo design is both an art and a science. The place to start with your design process is to understand all of the places where the current logo is used, what we need it to be communicating, why the current logo isn't meeting the communication goal, and what logo concepts might better meet that communication goal. Typical logo design processes generate 3-5 alternative concepts, get community feedback, and iterating until convergence, then generating final polished assets.

One soft constraint: we use a Unicode emoji in the Noto Color Emoji font because of accessibility, as it supports multilingual description of the chat bubble. Using an image compromises those accessibility features, as well as adds to performance issues, as we'd have to have many images of many different sizes, each with multilingual image descriptions.

We also chose the chat bubble because it aligns with the name of the `Phrase` output, which is the central output of the platform. So changing the logo breaks that alignment.

Another constraint: this appears very small in most uses, so it would have to be legible at very small image sizes.

**Comment by amyjko** [comment]
Hi! I'm Amy's GitHub bot. @rameenniazii95-star — Are you still working on this issue? Can you post a status update, and if you won't finish it, can you unassign yourself? We'll unassign you in a week if we don't get a reply.

**Comment by amyjko** [comment]
Hi, Amy's GitHub bot here. We didn't hear back from @rameenniazii95-star, so unassigning.
